api-lib/helpers/contributions.ts

```diff
@@ -0,0 +1,64 @@
+import faker from 'faker';
+
+import {
+  createContributionSchemaInput,
+  deleteContributionInput,
+} from '../../src/lib/zod';
+import { GraphQLTypes } from '../gql/__generated__/zeus';
+
+type InsertContributionInput = Partial<
+  typeof createContributionSchemaInput['_type']
+>;
+
+type DeleteContributionInput = typeof deleteContributionInput['_type'];
+
+type ContributionResult = Pick<
+  GraphQLTypes['contributions'],
+  'id' | 'description' | 'deleted_at' | 'user_id'
+>;
+
+export async function createContribution(
```

Comment:
I almost wonder if we could re-use this functions in our app, beyond jsut testing. And also, if they're just for testing, then if `api-lib/helpers` is the right place for them as I can imagine it being unclear this is a helper ONLY for test.

Reply:
You're right, moved them under `/api-test` with https://github.com/coordinape/coordinape/pull/1486/commits/cb825d4de93ba620e94d71fb76d22d6cc3efcd50